src/components/tooltip/tooltip.module.scss

```
@@ -12,7 +12,16 @@
   width: max-content;
   z-index: 1;
 
-  .triangle {
+  .triangle_bottom {
```

```suggestion
  .triangleBottom {
```
Very much a nit, but I think other classes in this repo use camelCase for classes